lib/wherebot.rb

```diff
@@ -88,11 +88,11 @@ def create
         destroy_message
         true
       else
-        Rails.logger.error e
+        Raven.captureException(e)
```

Comment:
`e` doesn't exist here. If you want to get an exception (caught below) when validations fail you can use `save!` instead.

Reply:
oh yeah!